tracer/src/Datadog.Trace.Trimming/Datadog.Trace.Trimming.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Version>2.32.0</Version>
+    <Version>2.32.0-prerelease</Version>
```

Comment:
I don't think this will work 🤔 When we run the next version bump, this will strip the `-prerelease` suffix, and replace it with `2.33.0`. I think we need to do the following instead:

```suggestion
    <Version>2.32.0</Version>
    <Version>$(Version)-prerelease</Version> <!-- Unconditionally add a prerelease suffix to the package, but keep the rest of the name so we can associate it with the rest of the release -->
```

Reply:
done in [0617805](https://github.com/DataDog/dd-trace-dotnet/pull/4246/commits/06178058379e730505e9c839e6ef4254c8bcf5be)